### ISSUE TITLE
perform_convergence bug

### ISSUE THREAD

**Issue by scottcoughlin2014** [COSMIC-PopSynth/COSMIC#364]
```
def perform_convergence(conv_params, conv_1, conv_2, log_file):
    """Performs the convergence calculations for each convergence parameter
       and binary state

       Parameters
       ----------
       conv_params : dict
           List of user supplied convergence parameters
       conv_1 : DataFrame
           Cumulative data set of conv arrays
       conv_2 : DataFrame
           Most recent data set of conv array from most recent BSE run
       log_file : file write
           File to log matches or if the convergence params are not appropriate
           e.g. eccentricity for a disrupted system

       Returns
       -------
       match : array
           Matches for each cumulative data set
    """
    match_all = []
    for conv_param in conv_params:
        if (conv_param == 'ecc') and (np.all(conv_1[conv_param] 1e-7)) and (np.all(conv_1[conv_param]=0.0)) and (bin_state == 0):

```
where in the perform_convergence is bin_state supplied? I think this is just a bug straightup that first conditonal does not make sense

**Comment by scottcoughlin2014** [comment]
@katiebreivik Camille ran into this bug a while ago and we never fix the logic of this if statement.

**Comment by katiebreivik** [comment]
Oh yeah! I remember talking about this. Will try to get a PR in before the call today

**Comment by katiebreivik** [comment]
Fixed by PR #365
